x-pack/plugins/observability/server/lib/rules/threshold/threshold_executor.test.ts

```diff
@@ -1394,7 +1394,7 @@ describe.skip('The metric threshold alert type', () => {
       await execute(true);
       const recentAction = mostRecentAction(instanceID);
       expect(recentAction.action).toEqual({
-        alertDetailsUrl: 'http://localhost:5601/app/observability/alerts/mock-alert-uuid',
+        alertDetailsUrl: '',
```

Comment:
Can we also remove the skip from this test?

Reply:
That test needs a lot more work before we are ready to remove the `skip` statement. It's failing beyond just the single test this PR is fixing.

Reply:
I thought so, here is the ticket for it:
https://github.com/elastic/kibana/issues/162299